### PR TITLE
DBZ-1067 Add column blacklist field to RelationalDatabaseConnectorConfig

### PR DESCRIPTION
**Pull request by grzegorz8**
https://issues.jboss.org/browse/DBZ-1067

### CODE REVIEW

**Comment by grzegorz8** [comment]
@gunnarmorling Done.

File: debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java [hunk 1]
```diff
@@ -137,6 +137,18 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withValidation(Field::isBoolean)
             .withDescription("Flag specifying whether built-in tables should be ignored.");
 
+    /**
+     * A comma-separated list of regular expressions that match fully-qualified names of columns to be excluded from monitoring
+     * and change messages. Fully-qualified names for columns are of the form {@code <databaseName>.<tableName>.<columnName>}
+     * or {@code <databaseName>.<schemaName>.<tableName>.<columnName>}.
```
[the remark this answers is not in the document]
Comment (grzegorz8) [reply debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java]: Yes, that'is right. How the comment should look like? In fact the comment is right - it's either with databaseName or without. We may add "depending on connector".

Nonetheless, the connector-specific documentations remain unchanged and correct.

[the remark this answers is not in the document]
Comment (gunnarmorling) [reply debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java]: >  it's either with databaseName or without.

Postgres has the notion of "schema", too.

> We may add "depending on connector".

Yes, something along these lines would be nice.

> Nonetheless, the connector-specific documentations remain unchanged and correct.

👍

[the remark this answers is not in the document]
Comment (gunnarmorling) [reply debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java]: Thx!

**Comment by gunnarmorling** [comment]
> And could you author the commit using an e-mail address linked to your GH account? Thanks!

Could you do that once more? Sorry to be such a pain about that, but I'd like to keep all commits properly attributable in the history. Once that's done it's good to go. Thanks again!

**Comment by grzegorz8** [comment]
Sorry, for that. Author has been fixed :)

**Comment by gunnarmorling** [comment]
Applied; thanks a lot, @grzegorz8!

**Pull request merged**